dev/tests/api-functional/testsuite/Magento/GraphQl/Customer/CreateCustomerTest.php

```diff
@@ -116,14 +116,18 @@ public function testCreateCustomerAccountWithoutPassword()
      */
     public function testCreateCustomerIfInputDataIsEmpty()
     {
+        $exceptionMessage = 'Field CustomerCreateInput.email of required type String! was not provided.
```

Comment:
a string either has "\n" or we can't do multi line strings like this. We usually don't add multi line exceptions